pkg/api/v1/schema.go

```diff
@@ -810,6 +810,9 @@ type DHCPOptions struct {
 	// If specified will pass option 66 to interface's DHCP server
 	// +optional
 	TFTPServerName string `json:"tftpServerName,omitempty"`
+
+	// Extra DHCP options
```

Comment:
Please add some more details here about what the valid code range is. Also, we'll need '+optional' too. Our api generates information from these comments.